{"uuid": "3b697b11-5a2b-45e5-a91e-01cbe42fe8ec", "children": ["c058fd1c-72b3-4603-b45f-1574cb36001c"], "befores": [{"name": "login", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/regular/tests/test_selenium_elemental.py\", line 8, in login\n    return login_page.LoginPage(driver)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/regular/pages/login_page.py\", line 16, in __init__\n    assert self._is_displayed(self._login_form)\n"}, "start": 1614289477829, "stop": 1614289478852}], "start": 1614289477829, "stop": 1614289478856}